src/sgp/SGP.cc

```diff
@@ -342,15 +349,35 @@ int main(int argc, char* argv[])
 		}
 	}
 
+	#ifdef __ANDROID__
+	JNIEnv* jniEnv = (JNIEnv*)SDL_AndroidGetJNIEnv();
+	if (jniEnv == NULL) {
+		SLOGE("Failed to get jni env");
+	}
+	RustPointer<char> configFolderPath(EngineOptions_getStracciatellaHome(jniEnv));
```

Comment:
Can we have something like `InitJNI` function at Rust side, then the Rust side remembers the jniEnv pointer. That way we don't need to have whole set of new functions or pass the pointer every call?

Reply:
That would be nice, but unfortunately it is not that easy as in rust the JNIEnv has an internal lifetime. But I'll take another shot at this when refactoring the Android asset VFS layer...